### DISCUSSION TITLE
Can alpine "timeout " due to inactivity?

### DISCUSSION THREAD

**Discussion by yabdab** [alpinejs/alpine#1619] [1. Help]
Is it possible for alpine to time out and stop controlling the DOM due to no user input ?

I discovered , on accident, that if I leave my app open for a few minutes, that no actions work EXCEPT for console.logs

So clicking on things seems to be triggering functions, but nothing is done tot he DOM.

I am using v3 currently, but I think this happened in v2.8 as well.

**Comment by KevinBatdorf** [comment]
Any errors in the browser console?

**Reply by yabdab** [reply]
No errors.  Clicks trigger debug logging in the app,  but visually the page does nothing.

**Reply by KevinBatdorf** [reply]
Is it possible to set up a demo or share some code?

**Reply by yabdab** [reply]
Eventually, but not yet. I guess I will revisit this later if  no one else has seem this.
